src/fetchers/stats-fetcher.js

```diff
@@ -118,9 +118,9 @@ async function fetchStats(
   // normal commits
   stats.totalCommits = user.contributionsCollection.totalCommitContributions;
 
-  // if include_all_commits then just get that,
+  // if NOT include_all_commits then just get that,
   // since totalCommitsFetcher already sends totalCommits no need to +=
-  if (include_all_commits) {
+  if (!include_all_commits) {
```

Comment:
If include_all_commits is true.. we should fetch all of the commits since user joined github.

Reply:
But that is already done above, isn't it? So the code is fetching the same info again.

After I modified the code, `/api/?username=aboqasem&include_all_commits=true` renders the expected output:

<img width="400" alt="Screenshot 2021-01-08 at 8 42 08 AM" src="https://user-images.githubusercontent.com/62098043/103960567-6748e080-518d-11eb-8de9-c3c726b8ed4a.png">

As well as `http://localhost:3000/api/?username=aboqasem`:

<img width="400" alt="Screenshot 2021-01-08 at 8 44 29 AM" src="https://user-images.githubusercontent.com/62098043/103960705-babb2e80-518d-11eb-8834-49653448c033.png">

Could you explain further what am I missing here?
Thank you!